src/conftest.py

```diff
@@ -9,7 +9,8 @@
 
 import pytest
 from pydantic import BaseModel, Field
-from pydantic.utils import deep_update
+from pydantic.v1 import ConfigDict
```

Comment:
Why do we need this, what is the v2 way of doing things?

Reply:
Hmm weird. According to the [docs](https://docs.pydantic.dev/latest/usage/model_config/) you can directly import it from `pydantic`:

```python
from pydantic import BaseModel, ConfigDict

class Model(BaseModel):
    model_config = ConfigDict(str_max_length=10)
```

Maybe I used the auto import fix in pycharm.